src/main/java/com/attacktimer/AttackTimerMetronomePlugin.java

```diff
@@ -339,11 +339,6 @@ public boolean isAttackCooldownPending()
     @Subscribe
     public void onChatMessage(ChatMessage event)
     {
-        if (event.getType() != ChatMessageType.SPAM)
-        {
-            return;
-        }
```

Comment:
I was noticing some eats were being skipped by this code block and not incrementing the counter, I think they may have change the filtering settings.